scripts/servers.py

```diff
@@ -677,10 +718,18 @@ def managed_protractor_server(
         del os.environ['MOBILE']
 
 
+# Here we use type Any because the argument 'kwargs' can accept an arbitrary
+# number of keyword arguments with different types of values.
 @contextlib.contextmanager
 def managed_webdriverio_server(
-        suite_name='full', dev_mode=True, debug_mode=False,
-        sharding_instances=1, chrome_version=None, mobile=False, **kwargs):
+    suite_name: str = 'full',
+    dev_mode: bool = True,
+    debug_mode: bool = False,
+    sharding_instances: int = 1,
+    chrome_version: Optional[str] = None,
+    mobile: bool = False,
+    **kwargs: Any
```

Comment:
No, because MyPy automatically infers the type of kwargs. For eg: if we type kwargs as 
`**Kwargs: str` then kwargs have type `Dict[str, str]` and if type `**Kwargs: Dict[str, Any]` then MyPy assume `kwargs` type as `Dict[str, Dict[str, Any]]`. Basically, we have to provide value type only ( not key type ).

Reply:
Ah, okay, thanks!